install-and-configure/advanced-configuration/gpu.md

```diff
@@ -0,0 +1,376 @@
+# NVIDIA GPU Monitoring Configurations
+
+## Monitoring GPU utilization
+
+In order for Kubecost to understand GPU utilization, Kubecost depends on metrics being available from NVIDIA [DCGM Exporter](https://github.com/NVIDIA/dcgm-exporter). Although Kubecost ships by default with GPU metrics being enabled, since DCGM Exporter is the provider of those metrics it is a required component when GPU monitoring is used with Kubecost and must be installed if it is not already. In many cases, DCGM Exporter may already be installed in your cluster, for example if you currently monitor NVIDIA GPUs with other software. But if not, follow the below instructions to install and configure DCGM Exporter on each of your GPU-enabled clusters.
```

Comment:
the part about 'kubecost ships by default w/ gpu metrics' but then 'dcgm is the provider of those metrics' is kinda confusing to me. It begs the question: what happens/what would I see by default if dcgm is not installed.

Reply:
Maybe I could rephrase this, something like
> Kubecost will search for GPU metrics by default, but since DCGM Exporter is the provider of those metrics it is a required component when GPU monitoring is used with Kubecost and must be installed if it is not already.

Reply:
yea i think thats better